portal/templates/admin/patients_by_org.html

```diff
@@ -20,6 +20,7 @@ <h4 class="tnth-headline">{% trans %}Patient List{% endtrans %}</h4>
               <input type="hidden" name="reset_cache" id="reset_cache" value="True"/>
               <input id="csrf_token" name="csrf_token" type="hidden" value="{{csrf_token()}}">
               <button type="submit" class="button btn btn-tnth-primary">{{_("Refresh Patient List")}}</button>
+              <span class="smaller-text byline-text">{{_("Last updated %(minuto)d minutes ago", minuto=qb_status_cache_age)}}</span> <!-- qb_status_cache_age should be calculated via view-->
```

Comment:
The placeholder string's meaning could be made more obvious (to translators).
```suggestion
              <span class="smaller-text byline-text">{{_("Last updated %(minutes)d minutes ago", minutes=qb_status_cache_age)}}</span> <!-- qb_status_cache_age should be calculated via view-->
```

Reply:
thanks, ivan,  fixed.